chroniker/tests/urls.py

```diff
@@ -16,17 +10,22 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.contrib import admin
+try:
+    from django.urls import include, re_path
+except ImportError:
+    from django.conf.urls import include, url as re_path
+
 
 admin.autodiscover()
 
 try:
     _patterns = [
-        url(r'^admin/', include(admin.site.urls)),
+        re_path(r'^admin/', include(admin.site.urls)),
     ]
 except ImproperlyConfigured:
     # Django >= 2.1.7.
     _patterns = [
-        url(r'^admin/', admin.site.urls),
+        re_path(r'^admin/', admin.site.urls),
```

Comment:
For consisteny with admin, should maybe have left these as `url` as well. Or, persisted with my earlier change from `url` to `re_path` in the admin as well (which I reverted).